packages/astro/components/Code.astro

```diff
@@ -43,7 +43,7 @@ function repairShikiTheme(html: string): string {
 }
 
 const highlighter = await shiki.getHighlighter({ theme });
-const _html = highlighter.codeToHtml(code, lang);
+const _html = highlighter.codeToHtml(code, { lang });
```

Comment:
Curious why this changed. Was this breaking? Or is this an optional way to pass the lang?

Reply:
Looks like they are both valid. I am fine with this change given that.

https://github.com/shikijs/shiki/blob/f462aa9bc0fcacd2e86513e7efb267aaa6e17afe/packages/shiki/src/highlighter.ts#L127-L137

Reply:
It's not breaking, it's just deprecated: https://github.com/shikijs/shiki/blob/e40647c00a036c14ae5e28dfde007f59a6485c37/packages/shiki/src/types.ts#L36